### PR TITLE
feat: added support for video bandwidth from getStats()

### PR DESCRIPTION
**Pull request by dnovicki**
*Issue #, if available:*
Adds extra field in chrome for video bandwidth through getStats()

*Description of changes:*
Added extra call to get BWEForVideo on chrome and add it to existing video prop in getStats()

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/js/rtp-stats.js [hunk 1]
```diff
@@ -5,86 +5,123 @@
...
+                        transmitBitrate:        statsReport.googTransmitBitrate
+                    }, statsReport.type, streamType);
                 }
             }
         }
     }
+    return extractedStats ? extractedStats : null;
+}
 
-    return extractedStats ? new MediaRtpStats(extractedStats, statsReport.type, streamType) : null;
```
Comment (plyvo) [inline src/js/rtp-stats.js]: I'd say we go with the original return because it follows DRY.

File: src/js/rtp-stats.js [hunk 2]
```diff
@@ -5,86 +5,123 @@
...
                         packetsLost:        statsReport.packetsLost,
                         packetsCount:       statsReport.packetsReceived,
                         audioLevel:         when_defined(statsReport.audioInputLevel),
-                        rttMilliseconds:    streamType === 'audio_ouptut' || streamType === 'video_output' ? when_defined(statsReport.roundTripTime) : null,
+                        rttMilliseconds:    streamType === 'audio_output' || streamType === 'video_output' ? when_defined(statsReport.roundTripTime) : null,
                         jbMilliseconds:     streamType === 'audio_output' || streamType === 'video_output' ? when_defined(statsReport.jitter, 0) * 1000 : null
-                    };
+                    }, statsReport.type, streamType);
+                }
+            } else if(statsReport.type === 'VideoBwe') {
```
Comment (plyvo) [inline src/js/rtp-stats.js]: The *if* is missing a space here and below.

File: src/js/rtp-stats.js [hunk 3]
```diff
@@ -5,86 +5,123 @@
...
     }
+    return extractedStats ? extractedStats : null;
+}
 
-    return extractedStats ? new MediaRtpStats(extractedStats, statsReport.type, streamType) : null;
+/**
+* Basic statistics object, represents core properties of any statistic
+*/
+class BaseStats {
+    constructor(paramsIn, statsReportType, streamType) {
```
Comment (plyvo) [inline src/js/rtp-stats.js]: *nitpick* `constructor(params = {}, statsReportType, streamType) {` would remove the need for the `const params = paramsIn || {};` below.

File: src/js/rtp-stats.js (hunk 1 above)
Comment (dnovicki) [reply src/js/rtp-stats.js]: The defined class is not always the same so I opted out of this pattern

File: src/js/rtp-stats.js (hunk 2 above)
Comment (dnovicki) [reply src/js/rtp-stats.js]: good catch

File: src/js/rtp-stats.js (hunk 1 above)
Comment (plyvo) [reply src/js/rtp-stats.js]: Just noticed the noticed `MediaBWEForVideoStats` type.

**Review by plyvo — COMMENTED** [review]
Have not tested but code changes look good overall. Only had nitpick comments.

**Pull request merged**